pkg/interface/link/src/js/components/settings.js

```diff
@@ -135,9 +132,9 @@ export class SettingsScreen extends Component {
                   props.resourcePath,
                   props.groupPath,
                   state.title,
-                  props.resource.metadata.description,
-                  props.resource.metadata['date-created'],
-                  uxToHex(props.resource.metadata.color)
+                  resource.metadata.description,
+                  resource.metadata['date-created'],
```

Comment:
Has any sanity checking been done on the contents of resource before we use these unsafe accessors to peer into the object?

Reply:
There's a catch for "if we loaded and the resource prop is empty or false, go to /~link" but if it wasn't a complete object it would crash on didMount, didUpdate, and on the render. I'll see what we can do. Option chaining ... alas